xrpl/models/requests/nfts_by_issuer.py

```diff
@@ -5,28 +5,22 @@
 from dataclasses import dataclass, field
 from typing import Any, Optional, Union
 
-from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.requests.request import LookupByLedgerRequest, Request, RequestMethod
 from xrpl.models.required import REQUIRED
 from xrpl.models.utils import require_kwargs_on_init
 
 
 @require_kwargs_on_init
 @dataclass(frozen=True)
-class NFTsByIssuer(Request):
+class NFTsByIssuer(Request, LookupByLedgerRequest):
     """
     The `nfts_by_issuer` method retrieves all of the NFTokens
     issued by an account
     """
 
     method: RequestMethod = field(default=RequestMethod.NFTS_BY_ISSUER, init=False)
     issuer: str = REQUIRED  # type: ignore
-    """
-    This field is required.
-    :meta hide-value:
-    """
```

Comment:
this comment block is still needed as it describes the issuer param. You also should make it more descriptive, something like "The unique identifier for an account"